src/components/BillCategoryTag/BillCategoryTag.svelte

```diff
@@ -0,0 +1,16 @@
+<script lang="ts">
+	import { Tag } from 'carbon-components-svelte';
+
+	export let isLarge = false;
+	let label = 'ขนส่งสาธารณะ';
+	let tagStyle = 'text-text-10 border-solid bg-white m-0 whitespace-nowrap';
+
+	let className = '';
+	export { className as class };
+</script>
+
+<Tag
+	class="{tagStyle} font-{isLarge ? 'semibold' : 'small'} {className}"
```

Comment:
Tailwind prune (remove) unused class when we do a production build. So if we split one class into several parts, Tailwind can't detect it and might remove the corresponding style. We must use `{isLarge ? 'font-semibold' : 'font-small'}` instead.